tests/behaviour/typeql/language/steps.rs

```diff
@@ -400,4 +402,9 @@ generic_step_impl! {
             matched entries of given {actual_answers}."
         );
     }
+
+    #[step(expr = "set time-zone is: {word}")]
```

Comment:
Let's move this to a fil called `//test/behaviour/util/steps.rs`, and then the current `util.rs` will become `//test/behaviour/util/mod.rs` to conform to Rust requirements. @dmitrii-ubskii  can help you with this further